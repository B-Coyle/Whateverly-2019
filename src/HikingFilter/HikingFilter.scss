.hikeCardContainer{
    margin: 5px;
    display: grid;
    grid-template-columns: 25% 50% 25%;
    grid-column-start: 2;
    grid-row-start: 2;
}