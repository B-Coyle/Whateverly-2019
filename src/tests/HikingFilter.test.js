import React from 'react';
import ReactDom from 'react-dom';
import HikingFilter from '../HikingFilter/HikingFilter.js';
import { shallow } from 'enzyme';

const mockCamping = [
    {
    "name": "Glacier Basin Campground",
    "location": "Estes Park",
    "season": "May-Sept",
    "elevation": 8500,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["Hiking", "Fishing", "Horseback Riding", "Wildlife"],
    "reservation-available": true,
    "nearest-city": null
      },
      {
    "name": "Grand Junction KOA",
    "location": "Grand Junction",
    "season": "All year",
    "elevation": 4600,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["ATVing"],
    "reservation-available": true,
    "nearest-city": null
    }
// ]

// describe('HikingFilter', () => {
//     let wrapper 
    
//     beforeEach(() => {
//         wrapper = shallow(
//             <HikingFilter />
//         )
//     });
    
//     it('should mock the snapshot', () => {
//         expect(wrapper).toMatchSnapshot();
//     });
,