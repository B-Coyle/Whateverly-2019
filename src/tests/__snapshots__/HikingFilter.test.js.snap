// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HikingFilter should mock the snapshot 1`] = `undefined`;
